server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -928,52 +845,6 @@ public File getHomeDir() {
         return homeDir;
     }
 
-    /**
-     * <strong>A filesystem location that has no real meaning and should not be regarded as providing any sort of useful
-     * information.</strong> The "modules" directory is the default location from which JBoss Modules looks to find
-     * modules. However, this behavior is in no way controlled by the value returned by this method, nor is it
-     * guaranteed that
-     * modules will be loaded from only one directory, nor is it guaranteed that the "modules" directory will be one
-     * of the directories used. Finally, the structure and contents of any directories from which JBoss Modules loads
-     * resources is not something available from this class. Users wishing to interact with the modular classloading
-     * system should use the APIs provided by JBoss Modules.
-     *
-     * @return a file
-     *
-     * @deprecated has no reliable meaning
-     */
-    @Deprecated
-    public File getModulesDir() {
-        return modulesDir;
-    }
-
-    /**
-     * Gets the directory under which OSGi bundles should be located.
-     *
-     * <p>
-     * Defaults to {@link #getHomeDir() homeDir}/bundles</p>
-     *
-     * @return the bundles directory
-     */
-    public File getBundlesDir() {
-        return bundlesDir;
-    }
-
-    private void configureBundlesDir(String dirPath, Properties providedProperties) {
-        boolean haveDirProperty = dirPath != null;
-        File tmp = getFileFromPath(dirPath);
-        if (tmp == null) {
-            if (haveDirProperty) {
-                throw ServerLogger.ROOT_LOGGER.bundlesDirectoryDoesNotExist(new File(dirPath).getAbsoluteFile());
```

Comment:
ServerLogger.ROOT_LOGGER.bundlesDirectoryDoesNotExist can be commented out after this.

Reply:
Thanks @yersan -- I commented that out. Some more metaspace saved!